client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -131,109 +145,136 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
-            <Form.Input
-              required
-              error={isFieldError(title)}
-              label="Title"
-              value={title}
-              onChange={(e, { value }) => setTitle(value)}
-              size="small"
+    <div>
+      <Toaster></Toaster>
+      <Modal
+        trigger={<Button className="default-btn" content="Submit a Pitch" />}
+        open={isOpen}
+        className="submit-modal"
+        onOpen={() => setIsOpen(true)}
+        onClose={() => setIsOpen(false)}
+        {...rest}
+      >
+        <Modal.Header>
+          <span style={{ paddingRight: '10px' }}>Submit a pitch</span>
+          <Popup
+            content={HelperMessage()}
+            trigger={<Icon size="small" name="info circle" />}
+            position="bottom left"
+            wide="very"
+            hoverable
+          />
+        </Modal.Header>
+        <Modal.Content>
+          <Form id="submit-pitch" onSubmit={submitPitch}>
+            <Form.Group widths="equal">
+              <Form.Input
+                error={isFieldError(title)}
+                label={<h5 className="label">Pitch Title</h5>}
+                value={title}
+                onChange={(e, { value }) => setTitle(value)}
+              />
+            </Form.Group>
+            <Form.Group widths="equal">
+              <Form.Input
+                error={isFieldError(link)}
+                label={<h5 className="label">Google Doc Link</h5>}
+                value={link}
+                onChange={(e, { value }) => setLink(value)}
+              />
+            </Form.Group>
+            <Form.TextArea
+              maxLength={MAX_LENGTH}
+              value={description}
+              onChange={(e, { value }) => setDescription(toString(value))}
+              label={<h5 className="label">Description</h5>}
+              error={isFieldError(description)}
             />
-            <Form.Input
-              required
-              error={isFieldError(link)}
-              label={'Link to Google Doc'}
-              value={link}
-              onChange={(e, { value }) => setLink(value)}
-              size="small"
+            <h5 className="label-topics">
+              Associated Interests
+              <p className="unbold">(select 1-4 interests) </p>
+            </h5>
+            <InterestsSelect
+              values={interests}
+              onChange={(values) => {
+                if (values.length <= 4) {
+                  setInterests(values.map((item) => item.value));
+                } else {
+                  interestNotify();
+                }
+              }}
             />
-          </Form.Group>
-
-          <Form.TextArea
-            required
-            maxLength={MAX_LENGTH}
-            value={description}
-            onChange={(e, { value }) => setDescription(toString(value))}
-            label="Description"
-            error={isFieldError(description)}
-          />
+            {user.role === 'WRITER' ? (
```

Comment:
This will not work. `Writer` is not a team name that is stored. `Writing` is. However, these are teams. Not roles. A role is a `CONTRIBUTOR`, `STAFF`, or `ADMIN`.